easybuild/easyconfigs/t/Trinity/Trinity-2.15.1-foss-2023a-Java-17.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     Custom
+#
+# Notes::
+##
+
+name = 'Trinity'
+version = '2.15.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://trinityrnaseq.github.io'
+description = """Trinity represents a novel method for the efficient and robust de novo reconstruction
+ of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
+ Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/releases/download/%(name)s-v%(version)s']
+sources = ['trinityrnaseq-v%(version)s.FULL.tar.gz']
+patches = ['Trinity-%(version)s_fix-bamsifter.patch']
+checksums = [
+    {'trinityrnaseq-v2.15.1.FULL.tar.gz': 'ba37e5f696d3d54e8749c4ba439901a3e97e14a4314a5229d7a069ad7b1ee580'},
+    {'Trinity-2.15.1_fix-bamsifter.patch': 'f374d446f06b8059a3b7cc57c286fc20d79b43ac222f48244ab83fb410997d1d'},
+]
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('CMake', '3.26.3'),
+]
+
+# for reference, list of dependencies in the container image used upstream:
+# https://github.com/trinityrnaseq/trinityrnaseq/blob/master/Docker/Dockerfile
+dependencies = [
+    ('Java', '17', '', SYSTEM),
+    ('ant', '1.10.12', '-Java-%(javaver)s', SYSTEM),
```

Comment:
I think `ant` is only used to build Java software, so it show probably not be a runtime dep
```suggestion
builddependencies = [
    ('ant', '1.10.12', '-Java-%(javaver)s', SYSTEM),
    ('Autotools', '20220317'),
    ('CMake', '3.26.3'),
]

# for reference, list of dependencies in the container image used upstream:
# https://github.com/trinityrnaseq/trinityrnaseq/blob/master/Docker/Dockerfile
dependencies = [
    ('Java', '17', '', SYSTEM),
```